### DISCUSSION TITLE
Run renovate in CI/CD in GitLab

### DISCUSSION THREAD

**Discussion by max-hammer** [renovatebot/renovate#33248] [Request Help] [answered]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitLab

### Please tell us more about your question or problem

A friend of mine is moving all his repos from Github to Gitlab, I should clarify how I can run dependabot for him. That's when I saw this project renovate-bot. I am very interested in this, I would like to learn how to solve the dependencies in each project with Giltab CI/CD and not with webhooks. In the documentation https://docs.renovatebot.com I did not find the answer to my question.

Dear community can you do this?

Which setup do I need? Which Environmnet Variables? If this is done via a CI/CD, are MRs/PRs also created?

**Comment by rarkins** [comment, marked as answer]
https://gitlab.com/renovate-bot/renovate-runner
